blog/_posts/2020-05-20-windows-support.md

```diff
@@ -0,0 +1,105 @@
+---
+slug: "windows-support"
+date: 2020-05-20
+author: nick
+layout: blog
+title: "Tilt Windows Support"
+image: "/assets/images/windows-support/windows.jpg"
+image_caption: "George Arents Collection, The New York Public Library. <em>Protecting your windows - a sandbag defence</em>.<br>Retrieved from <a href='http://digitalcollections.nypl.org/items/510d47da-9591-a3d9-e040-e00a18064a99'>http://digitalcollections.nypl.org/items/510d47da-9591-a3d9-e040-e00a18064a99</a>"
+tags:
+  - tilt
+  - windows
+---
+
+Historically, Tilt has supported multi-service development on Linux and macOS.
+
+Some of our users hacked it to work on Windows anyway. A few pro-active
+contributors even sent us PRs occasionally to fix Windows bugs ❤. Parts of
+Tilt worked well. Other parts didn't.
+
+So we're positively chuffed to announce full Windows support!
+
+## What Does That Mean Exactly?
+
+We're not saying we'll never have Windows bugs.
+
+We are saying:
+
+- Our core language examples in 
+  {[Python](https://github.com/tilt-dev/tilt-example-python), 
+   [NodeJS](https://github.com/tilt-dev/tilt-example-nodejs), 
+   [Go](https://github.com/tilt-dev/tilt-example-go), 
+   [Java](https://github.com/tilt-dev/tilt-example-java), 
+   [C#](https://github.com/tilt-dev/tilt-example-csharp)} 
+  work on Windows.
+- Every Tilt change runs our tests on Windows.
+- We maintain binaries and installers for Windows for [every Tilt release](https://github.com/tilt-dev/tilt/releases).
+
+In other words, Windows is now a first-class citizen alongside Linux and macOS.
+
+Install it with:
+
+```
```

Comment:
Can we make this use powershell syntax highlighting? I'm honestly not sure.

Reply:
the answer is yes!